Make.bat

```diff
@@ -20,10 +20,10 @@ ECHO.
 ECHO.Usage:
 ECHO.    .\make env     Switches to a Python virtual environment
 ECHO.    .\make run     Runs Jupyter tutorials
-ECHO.    .\make lint    Runs Python source code analisys tool
+ECHO.    .\make lint    Runs Python source code analysys tool
```

Comment:
```suggestion
ECHO.    .\make lint    Runs Python source code analysis tool
```

Reply:
@lcapelluto Thank you for pointing it out. I have just addressed it.